apps/web/src/hooks/useThemeChange.tsx

```diff
@@ -0,0 +1,42 @@
+import { ColorScheme, useMantineColorScheme } from '@mantine/core';
+import { IconDarkMode, IconLightMode, IconTonality } from '@novu/design-system';
+import { ColorSchemePreferenceEnum, THEME_TITLE_LOOKUP, useLocalThemePreference, useSegment } from '@novu/shared-web';
+import { useEffect, useMemo } from 'react';
+import { useDebounce } from './useDebounce';
+
+type ThemeChange = { colorScheme: ColorScheme; themeStatus: string };
```

Comment:
```suggestion
type ThemeChange = { colorScheme: ColorScheme; themeStatus: ColorSchemePreferenceEnum };
```

Reply:
Good catch, missed it during refactoring 👍🏼